README.md

```diff
@@ -228,3 +228,24 @@ nvm exec 0.10 ./pm2 start servers_extra.json
 
 #### Run
 Once services have started, run `npm start` to open Firefox with a local profile. Access the Hello service as you normally would.
+
+### Running with MailDev
+
+If you want to inspect emails, you can run fxa-local-dev with MailDev.
+
+#### Install
+```bash
+npm install maildev -g
+```
+
+#### Run
+```bash
+./pm2 start servers_maildev.json
```

Comment:
Is this really required? ./pm2 start servers.json works fine, and we already have that documented.